mlir/include/mlir/Dialect/ControlFlow/IR/ControlFlowOps.td

```diff
@@ -233,7 +233,7 @@ def SwitchOp : CF_Op<"switch",
      Pure, Terminator]> {
   let summary = "switch operation";
   let description = [{
-    The `switch` terminator operation represents a switch on a signless integer
+    The `cf.switch` terminator operation represents a switch on a signless integer
```

Comment:
If we're going to fix the doc, can we fix `cond_br` above to be consistent as well?

Reply:
For sure. Should I fix the `cond_br` description and reopen PR? Moreover, the `assert` operation description must be fixed too.

Reply:
Fixed `assert` and `cond_br` descriptions.